cookiecutter/prompt.py

```diff
@@ -26,7 +26,7 @@ def prompt_for_config(context, no_input=False):
     env = Environment()
 
     for key, raw in iteritems(context['cookiecutter']):
-        val = env.from_string(raw).render(cookiecutter=cookiecutter_dict)
+        val = env.from_string(str(raw)).render(cookiecutter=cookiecutter_dict)
```

Comment:
How about an `isinstance` check first?
Unfortunately, it's a little bit more fiddly to be py2 and py3 compatible- I suggest adding something from [this SO question](http://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility) to `compat.py`